package.json

```diff
@@ -8,8 +8,8 @@
     "start-watch": "nodemon index.js",
     "lint": "standard",
     "pretest": "npm run lint",
-    "test": "jest",
-    "test-watch": "jest --watch"
+    "test": "jest --forceExit",
+    "test-watch": "jest --watch --forceExit"
   },
   "author": "",
```

Comment:
Not strictly part of this PR, but we really shouldn't be leaving the defaults in this file.
"name": "mine-support-calculation-service",
"description": "Digital service mock to claim public money in the event property subsides into mine shaft.",
"version": "0.0.1",
"homepage": "https://github.com/DEFRA/mine-support-calculation-service",
"author": "Defra DDTS",
"contributors": [
"John Watson john.watson2@rpa.gov.uk",
"Mark Harrop mark.harrop@defra.gov.uk",
"Matthew Collins matthew.collins@defra.gov.uk",
"David Bingham david.bingham@defra.gov.uk"
],
"license": "OGL-UK-3.0",

Reply:
Updated as suggested